app/components/pagination.gjs

```diff
@@ -34,6 +36,12 @@ import { eq } from 'ember-truth-helpers';
       data-test-pagination-next
     >
       {{svgJar 'right-pag'}}
+      {{#if (eq @pagination.currentPage @pagination.maxPages)}}
+        <Tooltip>
+          Page limit reached for performance reasons. For bulk data access, please visit
```

Comment:
I might massage this _very_ slightly into:

```suggestion
          For performance reasons, no more pages are available. For bulk data access, please visit
```

But I don't care very much either way. This is fine as it is.